sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/AzMonListExtensions.cs

```diff
@@ -239,6 +239,20 @@ internal static string GetDefaultDbPort(string? dbSystem)
             return target;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static string? GetTargetUsingServerAttributes(this AzMonList tagObjects, string defaultPort)
```

Comment:
Recommend adding tests for this method too, but not a blocker.

Reply:
Agree. The current test is covering multiple methods right now. Will take this in big refactor of test if that is ok.